src/main.ts

```diff
@@ -25,21 +25,21 @@ export async function run(softError = false): Promise<string> {
       await clone(ref, dir)
     }
 
-    core.info('Building the Lingua Franca compiler...')
-    await build(dir)
-
     configurePath(dir)
 
     core.info('Checking all Lingua Franca files:')
     skipDirs.push(dir)
-    if ((await checkAll('.', ignore)) === false) {
+    if ((await checkAll(dir, '.', ignore)) === false) {
```

Comment:
I don't think this one is right because `dir` refers to the directory of the `lingua-franca` checkout..

Reply:
I think we should parameterize it and have `.` as the default maybe..

Reply:
It needs to have access to the directory of the `lingua-franca` checkout so that it can run `lfc`

Reply:
But we put `lfc` on the path

Reply:
Let me do a little bit of experimentation. Thank you so much for fixing the build issues!